examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -145,6 +170,12 @@ def __init__(
             camera_helper=self.camera_helper,
         )
 
+        # Register UI callbacks
+        self.ui.on_pick.registerCallback(self._on_pick)
+        self.ui.on_place.registerCallback(self._on_place)
+        self.ui.on_open.registerCallback(self._on_open)
+        self.ui.on_close.registerCallback(self._on_close)
+
```

Comment:
<3

Reply:
To use these, would I need to pass on `ui` object to `LLMController`?

Reply:
You can probably just do it from `rearrange_v2` initialization code, which has access to both the UI and `LLMController`.

Something like this would work:
```
self._user_data[0].ui.on_pick.registerCallback(llm_controller.on_pick())
```

If we ever want to scale this to N users, we would just pass the user index in the event data. For now, this does the job.